oak/server/rust/oak_runtime/src/runtime/mod.rs

```diff
@@ -138,6 +138,12 @@ pub struct Runtime {
     next_node_id: AtomicU64,
 }
 
+#[derive(Debug)]
+pub enum ErroneousChannelStatus {
+    HasInvalid(Vec<ChannelReadStatus>),
+    Err(OakStatus),
```

Comment:
Probably best to avoid `Err` as a name for this enum, it's too likely to make people assume that the other variant is `Ok`.